homeassistant/components/vulcan/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "vulcan",
+  "name": "Uonet+ Vulcan",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/vulcan",
+  "requirements": ["vulcan-api==2.0.3"],
+  "dependencies": [],
+  "codeowners": ["@Antoni-Czaplicki"],
+  "iot_class": "cloud_polling",
+  "quality_scale": "platinum"
```

Comment:
This integration doesn't reach platinum level. Complete test coverage is missing for one. Please remove the quality scale for now.